cellmlmanip/model.py

```diff
@@ -555,21 +558,29 @@ def get_symbol_by_ontology_term(self, namespace_uri, local_name):
             raise ValueError('Multiple variables annotated with {%s}%s' %
                              (namespace_uri, local_name))
 
-    def _get_symbols_by_rdf(self, predicate, object_=None):
+    def get_rdf_annotations(self, subject=None, predicate=None, object=None):
+        """Searches the RDF graph and returns the triples for the given (subject, predicate, object)"""
+        subject = create_rdf_node(subject)
+        predicate = create_rdf_node(predicate)
+        object = create_rdf_node(object)
+        return self.rdf.triples((subject, predicate, object))
+
+    def get_rdf_value(self, subject, predicate):
+        """Get the value of an RDF object connected to ``subject`` by ``predicate``."""
```

Comment:
Add to docstring (again, in para after blank line) that exactly one triple must match, and the resulting object should be a literal, the string value of which will be returned.